### PR TITLE
Update RIOT target to 2021.10

### PR DESCRIPTION
**Pull request by rtakacs**
* Updated RIOT target from `2020.01` to `2021.10`
  * Modified Makefile according to RIOT changes
* Updated README.md file mostly about missing serial communication information
* Removed Makefile.riot file and `tools/build.py` is used instead of that

### CODE REVIEW

File: targets/riot-stm32f4/README.md [hunk 1]
```diff
@@ -1,96 +1,139 @@
...
+  PA3 = RX Data
+  GND = GND
 ```
 
-This will create a hex file in the `targets/riot-stm32f4/bin` folder:
-```
-riot_jerry.elf
+The device should be visible as `/dev/ttyUSB0`.
+
+#### 5.1. Serial communication by `pyterm`
```
Comment (robertsipka) [inline targets/riot-stm32f4/README.md:80]: From previous experience, pyterm is often flakey. Please consider leaving only the minicom in the description.

File: targets/riot-stm32f4/README.md [hunk 2]
```diff
@@ -1,96 +1,139 @@
...
+#### 5.2. Serial communication by `minicom`
 
-#### 5. Cleaning
+Use `minicom` communication program with `115200` baud rate.
 
-To clean the build result:
-```
-make -f ./targets/riot-stm32f4/Makefile.riot clean
+```sh
+sudo minicom --device=/dev/ttyACM0 --baud=115200
```
Comment (robertsipka) [inline targets/riot-stm32f4/README.md:103]: it's ttyUSB0, not ttyACM0.

**Review by robertsipka — APPROVED** [review]
I've tried it on a stm32f4 board, it worked for me. Except for the minor comments, LGTM.

**Comment by rtakacs** [comment]
@robertsipka FYI, I've extended the Makefile with JerryScript compilation. In this case, RIOT automatically builds JerryScript as its dependency. (And that is not a different step for the user.)

**Review by dbatyai — APPROVED** [review]

**Pull request merged**